library/include/ck/library/tensor_operation_instance/gpu/grouped_conv_fwd/device_grouped_conv_fwd_xdl_scale_instance.hpp

```diff
@@ -22,8 +22,8 @@ using S = ck::Sequence<Is...>;
 
 using namespace ck::tensor_layout::convolution;
 
-using PassThrough = ck::tensor_operation::element_wise::PassThrough;
-using Scale       = ck::tensor_operation::element_wise::Scale;
+using PassThrough    = ck::tensor_operation::element_wise::PassThrough;
```

Comment:
This file should be not changed. You have to create the same file - device_grouped_conv_fwd_xdl_dynamic_op_instance.hpp. We can remove this file in next release.